app/views/lead_providers/partnership_csv_uploads/errors.html.erb

```diff
@@ -0,0 +1,53 @@
+<% content_for :title, "CSV results" %>
+<% content_for :before_content, govuk_back_link(text: "Back", href: new_lead_providers_report_schools_partnership_csv_uploads_path) %>
+<h1 class="govuk-heading-xl">Your CSV has errors</h1>
```

Comment:
Do we want the content do be in a grid?

Reply:
🤷 not sure. I went without because it's quite a wide table. Haven't been given any guidance on this

Reply:
I'll get some design review for this whole journey and apply any minor markups like this separately